dvc/remote/gdrive.py

```diff
@@ -79,6 +112,12 @@ class RemoteGDrive(RemoteBASE):
     GDRIVE_CREDENTIALS_DATA = "GDRIVE_CREDENTIALS_DATA"
     DEFAULT_USER_CREDENTIALS_FILE = "gdrive-user-credentials.json"
 
+    # Indicates usage of shared or default remote drive
+    corpora = "default"
```

Comment:
looks like a hack, and we don't actually need this I believe

Reply:
Removed, thanks